### DISCUSSION TITLE
Need a abort-while-test-going-on kinda feature.

### DISCUSSION THREAD

**Discussion by Xyerophyte** [monkeytypegame/monkeytype#4090] [Ideas] [answered]
Once you start the test, you will have to let the timer end if you just whiffed many times in a row and want to abort.
So I think there should be a quick restart/abort option/feature.

**Comment by Miodec** [comment, marked as answer]
<img width="739" alt="image" src="https://user-images.githubusercontent.com/13181393/226205660-390b3c32-3c21-4baa-aa48-217312490016.png">

Its been a thing since the very beginning of the site ;)
